src/components/GroupsTable/GroupsTable.cy.js

```diff
@@ -0,0 +1,216 @@
+/* eslint-disable camelcase */
+import { mount } from '@cypress/react';
+import {
+    changePagination,
+    checkEmptyState,
+    checkPaginationTotal,
+    checkPaginationValues,
+    checkTableHeaders,
+    CHIP,
+    CHIP_GROUP,
+    hasChip,
+    PAGINATION_VALUES,
+    SORTING_ORDERS,
+    TEXT_INPUT,
+    TOOLBAR,
+    TOOLBAR_FILTER
+} from '@redhat-cloud-services/frontend-components-utilities';
+import _ from 'lodash';
+import React from 'react';
+import { Provider } from 'react-redux';
+import { MemoryRouter } from 'react-router-dom';
+import fixtures from '../../../cypress/fixtures/groups.json';
+import { groupsInterceptors as interceptors } from '../../../cypress/support/interceptors';
+import { getStore } from '../../store';
+import GroupsTable from './GroupsTable';
+
+const ORDER_TO_URL = {
+    ascending: 'ASC',
+    descending: 'DESC'
+};
+
+const DEFAULT_ROW_COUNT = 50;
```

Comment:
@gkarat i remember @mkholjuraev raised a conversation about default pagination 20 - should we stick with it?